.github/workflows/import_pending_sql.yml

```diff
@@ -0,0 +1,32 @@
+name: import-pending-sql
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  import-pending-sql:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      - name: Import and commit pending sql
+        run: |
+          git config user.email "azerothcorebot@gmail.com" && git config user.name "AzerothCoreBot"
+          cd bin/
+          source acore-db-pendings
+          cd ..
+          git fetch --unshallow origin ${BRANCH}
+          git add -A .
+          git commit -am "Import pending SQL update file..." -m "Referenced commit(s):${GITHUB_SHA}" || true
+        env:
+          BRANCH: ${{ steps.extract_branch.outputs.branch }}
+      - name: Push changes
+        uses: ad-m/github-push-action@fe38f0a751bf9149f0270cc1fe20bf9156854365
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: ${{ steps.extract_branch.outputs.branch }}
```

Comment:
do we have a way to test this? or, I guess, we'll find out once it's on master?

Reply:
I already tested that on my own fork by making a dummy update file that was then imported.
https://github.com/Rochet2/azerothcore-wotlk/runs/580733452?check_suite_focus=true
https://github.com/Rochet2/azerothcore-wotlk/commit/f9a6041b1a576659d542a17c43a7509864d2d839
also it seems github has you token automatically .. so I didnt need to set it anywhere for it to work in this workflow. Unsure if I have already set some token before, but it would look like adding some secret env was not necessary.

Reply:
amazing!